drivers/pwm/Kconfig.mcux

```diff
@@ -2,10 +2,15 @@
 
 # Copyright (c) 2019 Linaro Limited.
 # SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 NXP
 
 config PWM_MCUX
 	bool "MCUX PWM driver"
 	default y
 	depends on DT_HAS_NXP_IMX_PWM_ENABLED
 	help
 	  Enable mcux pwm driver.
+
+if PWM_MCUX
+
+endif # PWM_MCUX
```

Comment:
why is this here?

Reply:
Originally not intended, however added when I needed a Kconfig option for the pwm enable-wait bit to be enabled. Due to a recent PR this Kconfig is no longer necessary but the if PWM_MCUX may still be needed for future PRs.

Reply:
I think this can be deleted